opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationCreation.tsx

```diff
@@ -141,6 +148,7 @@ export const OrganizationCreationForm: FunctionComponent<OrganizationFormProps>
         description: values.description,
         x_opencti_reliability: values.x_opencti_reliability,
         x_opencti_organization_type: values.x_opencti_organization_type,
+        x_opencti_score: parseInt(String(values.x_opencti_score), 10),
```

Comment:
It's strange to use `String()` here
```suggestion
        x_opencti_score: parseInt(values.x_opencti_score, 10),
```

Reply:
as `values.x_opencti_score` could be null or undefined, it needs to be transformed as string , see confidence line 152

Reply:
![image](https://github.com/user-attachments/assets/3850fbe5-e579-4a1a-809e-8a6eaccc6323)
Same issue will occur, either with String and parseInt maybe the best workaround would be to initialize (line 189) it  to 50 (like we do with Indicator creation score)

Reply:
we don't want score to be initialized as Indicator or Observable. we want it to be null when we open the creationForm.

Reply:
if I remove the String() I get this type error:
![image](https://github.com/user-attachments/assets/c216dc10-fc96-41c3-93f5-7e69bfd73b6b)

Reply:
Patched together :+1: